python/pyarrow/table.pxi

```diff
@@ -3403,6 +3403,8 @@ cdef class RecordBatch(_Tabular):
         ----------
         null_to_nan : bool, default False
             Whether to write null values in the result as ``NaN``.
```

Comment:
We should probably note somewhere that if `null_to_nan` is false and there are nulls then it will throw an error